console_bot.py

```diff
@@ -14,6 +14,11 @@ def print_help():
   add-birthday [name] [birthday] - Adds birthday to the contact
   show-birthday [name]           - Shows birthday for specific contact
   birthdays                      - Shows birthdays for all contacts celebrating next week
+  add-note
+  find-notes
+  delete-note
+  edit-note
+  all-notes
```

Comment:
Don't forget to add descriptions here

Reply:
Good point! I have some idea about showing help message, will introduce in separate PR and update the description